### PR TITLE
fixes for nosip_plugin

### PR DESCRIPTION
**Pull request by DmitryYudin**
Please, see the commits comment for details.

### CODE REVIEW

File: plugins/janus_nosip.c [hunk 1]
```diff
@@ -1186,6 +1186,11 @@ static void *janus_nosip_handler(void *data) {
 			const char *msg_sdp_type = json_string_value(json_object_get(generate ? msg->jsep : root, "type"));
 			const char *msg_sdp = json_string_value(json_object_get(generate ? msg->jsep : root, "sdp"));
 			gboolean sdp_update = json_is_true(json_object_get(generate ? msg->jsep : root, "update"));
+			if(!generate) {
+				if(session->media.ready) {
```
Comment (lminiero) [inline plugins/janus_nosip.c]: Why two layers of if when they can be together?

File: plugins/janus_nosip.c [hunk 2]
```diff
@@ -1988,7 +1993,7 @@ static void *janus_nosip_relay_thread(void *data) {
 			!g_atomic_int_get(&session->destroyed) && !g_atomic_int_get(&session->hangingup)) {
 		if(session->media.updated) {
 			/* Apparently there was a session update */
-			if(have_server_ip && (inet_aton(session->media.remote_ip, &server_addr.sin_addr) == 0)) {
+			if(have_server_ip && (inet_aton(session->media.remote_ip, &server_addr.sin_addr) != 0)) {
```
Comment (lminiero) [inline plugins/janus_nosip.c]: Aaah, good catch: I checked the SIP and SIPre plugins and they're already correct, so I guess it was either a copy/paste error at the time, or a failure to fix this one as well when we fixed those.

File: plugins/janus_nosip.c (hunk 1 above)
Comment (DmitryYudin) [reply plugins/janus_nosip.c]: single line - ok)

Comment (DmitryYudin) [reply plugins/janus_nosip.c]: force updated

**Comment by lminiero** [comment]
Thanks, merging then!

**Pull request merged**